autoPyTorch/pipeline/components/setup/network_backbone/base_network_backbone.py

```diff
@@ -52,12 +52,7 @@ def fit(self, X: Dict[str, Any], y: Any = None) -> BaseEstimator:
         self.check_requirements(X, y)
         X_train = X['X_train']
```

Comment:
Can we add a typing for this variable `X_train`?

Reply:
sure